kubespawner/spawner.py

```diff
@@ -1784,6 +1822,35 @@ def _start(self):
                 else:
                     raise
 
+
+        secret = self. get_secret_manifest()
+        if secret:
+            try:
+                yield self.asynchronize(
+                    self.api.create_namespaced_secret,
+                    namespace=self.namespace,
+                    body=secret
+                )
+            except ApiException as e:
+                if e.status == 409:
+                    self.log.info("Secret " + self.pod_name + " already exists, so did not create new secret.")
+                else:
+                    raise
+
+        service = self. get_service_manifest()
```

Comment:
```suggestion
        service = self.get_service_manifest()
```
Looks like a typo crept in here